Backend/app/auth/helpers.js

```diff
@@ -27,7 +26,8 @@ const helpers = {
   },
   async sendInviteLink(email, token) {
     const mailTransporter = await mailer();
-    const url = `http://${process.env.HOST}/signup/${token}`;
+    const url = `http://localhost:3006/signup/${token}`;
```

Comment:
Check my latest PR, don't hardcode this

Reply:
@python-plusplus

Reply:
@youssefsoli Yeah that was temp for the demo, my bad forgot to change it.  I'm merging with your changes rn so I already removed that and I'll accept incoming on changes so all these oversights should be buffed out